src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,160 @@
+  const history = useHistory();
+
+  const deleteValidationSchema = Yup.object().shape({
+    deleteMessage: Yup.string()
+      .min(3, t('unpublish_actions.unpublish_comment_min_length_feedback'))
+      .required(t('unpublish_actions.unpublish_comment_required')),
+  });
+  const handleDelete = async (values: DeleteForm) => {
+    const unpublishRequest = (): UnpublishPublicationRequest => {
+      return selectedDuplicate
+        ? {
+            type: 'UnpublishPublicationRequest',
+            duplicateOf: selectedDuplicate.identifier,
+            comment: values.deleteMessage,
+          }
+        : { type: 'UnpublishPublicationRequest', comment: values.deleteMessage };
+    };
+
+    setAwaitingUnpublishingResponse(true);
+
+    const unpublishingResponse = await unpublishRegistration(registration, unpublishRequest());
+    setAwaitingUnpublishingResponse(false);
+    if (isErrorStatus(unpublishingResponse.status)) {
+      dispatch(setNotification({ message: t('feedback.error.update_registration'), variant: 'error' }));
+    }
+    if (isSuccessStatus(unpublishingResponse.status)) {
+      setShowDeleteModal(false);
+      history.push(`/registration/${registration.identifier}?shouldNotRedirect`);
+    }
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem' }}>
+        <Divider />
+        <Breadcrumbs
+          sx={{ alignSelf: 'center' }}
+          itemsBeforeCollapse={0}
+          itemsAfterCollapse={0}
+          maxItems={0}
+          aria-label={t('common.show_more_options')}>
+          <Button
+            data-testid={dataTestId.unpublishActions.openUnpublishModalButton}
+            variant="outlined"
+            onClick={() => setShowDeleteModal(true)}>
+            {t('common.delete')}
+          </Button>
+        </Breadcrumbs>
+      </Box>
+      <Modal
+        dataTestId="delete-registration-modal"
+        headingText={t('registration.delete_registration')}
+        open={showDeleteModal}
+        onClose={() => setShowDeleteModal(false)}>
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+          <Box>
+            <RequiredDescription />
+          </Box>
+          <Box sx={{ gap: '1rem' }}>
+            <Typography variant="h3">{t('unpublish_actions.unpublish_registration')}</Typography>
+            <Typography>{t('unpublish_actions.unpublish_registration_detail_1')}</Typography>
+            <Typography>{t('unpublish_actions.unpublish_registration_detail_2')}</Typography>
+          </Box>
+
+          <Formik
+            initialValues={{
+              deleteMessage: '',
+            }}
+            validationSchema={deleteValidationSchema}
+            onSubmit={(values, _formikHelpers) => {
+              handleDelete(values);
+            }}>
+            {({ values, errors, touched, handleChange, handleBlur, handleSubmit }) => (
+              <form onSubmit={handleSubmit}>
```

Comment:
`onSubmit` er satt i `<Formik>` så den trengs ikke her, da den bare gir forvirring om hvilken som faktisk brukes. `<Form>` fra formik må kanskje brukes for at det skal virke da, når jeg tenker meg om..? Kan legge til en `noValidate` også, for å unngå å få både popup om at feltet mangler data (som browser gjør som default), i tillegg til custom feilmelding under feltet

```suggestion
              <Form noValidate>
```